src/apps/testapps/testCellsToLinkedMultiPolygon.c

```diff
@@ -297,4 +297,66 @@ SUITE(cellsToLinkedMultiPolygon) {
 
         H3_EXPORT(destroyLinkedMultiPolygon)(&polygon);
     }
+
+    TEST(kRingResolutions) {
+        // This is a center-face base cell, no pentagon siblings
+        H3Index baseCell = 0x8073fffffffffff;
+        H3Index origin = baseCell;
+
+        H3Index indexes[] = {0, 0, 0, 0, 0, 0, 0};
+        int numHexes = 7;
+
+        for (int res = 1; res < 15; res++) {
```

Comment:
Note that even with this fix, both of these tests fail at res 0. I'm not sure where the additional imprecision lies for that case, but I think we need vertex mode to fix it.